### PR TITLE
header - landing page

### PR DESCRIPTION
**Pull request by Aditiarzky**

### CODE REVIEW

File: resources/css/app.css [hunk 1]
```diff
@@ -1,3 +1,20 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+@import url('https://fonts.googleapis.com/css2?family=Bree+Serif&display=swap');
+
+.title-app{
```
Comment (Fikrianamm) [inline resources/css/app.css:7]: ganti, tambah fontfamily neng tailwindcss.config.js
theme: {
    extend: {
      fontFamily: {
        breeSerif: ["Bree Serif", "system-ui", "sans-serif"],
      },
    },
  },  
  implement neng className (font-breeSerif)

File: resources/views/app.blade.php [hunk 2]
```diff
@@ -7,13 +7,17 @@
         <title inertia>{{ config('app.name', 'Laravel') }}</title>
 
         <!-- Fonts -->
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.0/css/all.min.css">
         <link rel="preconnect" href="https://fonts.bunny.net">
         <link href="https://fonts.bunny.net/css?family=figtree:400,500,600&display=swap" rel="stylesheet" />
-
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Bree+Serif&display=swap" rel="stylesheet">
```
Comment (Fikrianamm) [inline resources/views/app.blade.php:15]: coba hapus link font soale ws mok import neng css haruse iso

Comment (Aditiarzky) [reply resources/views/app.blade.php:15]: tak bajal mou gak keno

Comment (Aditiarzky) [reply resources/views/app.blade.php:15]: sakjane gpp, kan nko lewat e neng app.blade.php kabeh, gak perlu ngeki link maneh

Comment (Fikrianamm) [reply resources/views/app.blade.php:15]: yo gpp cuma jadine nulis 2 kali

**Pull request closed without merging**